tracing-subscriber/tests/layer_filters/main.rs

```diff
@@ -47,23 +48,23 @@ fn basic_layer_filters() {
 }
 
 #[test]
-fn basic_layer_filters_spans() {
+fn basic_subscriber_filters_spans() {
```

Comment:
can we change this to use the v0.1.x names? e.g. `basic_subscriber_filters_spans`, as it was previously?